webapp/views/App/views/Analysis/Chain/Step/VariablesPreviousStep/VariablesPreviousStep.scss

```diff
@@ -0,0 +1,9 @@
+.processing-step__variables-previous-step-table {
+  height: 300px;
```

Comment:
why fixed height?

Reply:
because I've couldn't make it work quickly with a 100% height... ;)

Reply:
ok I'll check the ui once merged